### ISSUE TITLE
PHP Warning:  Undefined array key "HTTP_USER_AGENT"  "HTTP_HOST"

### ISSUE THREAD

**Issue by Riesuu** [derricksmith/phpsaml#80]
Hello,
I have a problem with my cron and during the debugging I have this error

```
root@glpi-tst-01:/tmp# php /var/www/glpi/bin/console glpi:task:unlock
PHP Warning:  Undefined array key "HTTP_USER_AGENT" in /var/www/glpi/plugins/phpsaml/setup.php on line 168
PHP Warning:  Undefined array key "HTTP_HOST" in /var/www/glpi/plugins/phpsaml/setup.php on line 197
root@glpi-tst-01:/tmp# php /var/www/glpi/bin/console glpi:task:unlock -t MailCollector::mailgate
PHP Warning:  Undefined array key "HTTP_USER_AGENT" in /var/www/glpi/plugins/phpsaml/setup.php on line 168
PHP Warning:  Undefined array key "HTTP_HOST" in /var/www/glpi/plugins/phpsaml/setup.php on line 197
```

glpi 9.5.7 phpsaml 1.1.3

thanks

**Comment by baplb** [comment]
Hello,

The following change in the setup.php, fix the problem for me :
https://github.com/derricksmith/phpsaml/issues/58#issuecomment-911918282

**Comment by derricksmith** [comment]
> Hello,
> 
> 
> 
> The following change in the setup.php, fix the problem for me :
> 
> https://github.com/derricksmith/phpsaml/issues/58#issuecomment-911918282
> 
> 
> 
> 

This issue is fixed in 1.2.0 branch.  The branch has not been merged.  I have a few more changes for this version.
